game.js

```diff
@@ -81,47 +52,52 @@ function checkCityInDB(chatID, city, letter) {
   return null;
 }
 
-function selectCityByLetter(chatID, letter, cities) {
-  let findCities = cities.filter(
-    item => item[0] == letter && !sessions[chatID].spentCities.has(item)
+async function selectCityByLetter(chatID, letter) {
+  console.log("Ищем город на букву " + letter);
+  let findCities = await places_api.findCitiesByLetter("город " + letter);
+  let result = findCities.filter(
+    item => item[0] === letter && !sessions[chatID].spentCities.has(item)
   );
-  if (findCities.length === 0) {
+  console.log(findCities);
+  if (result.length === 0) {
     return null;
   } else {
-    return randomCity(findCities);
+    return randomCity(result);
   }
 }
 
-function start(chatID) {
+async function start(chatID) {
   let result = { messages: [] };
   result.messages.push(
     "Начинаем игру. \nЯ называю город, а ты должен в ответ назвать любой город \nначинающийся на последнюю букву моего города.\nГорода не должны повторятся."
   );
+  let selectedCity = await selectCityByLetter(chatID, randomCity(alphabet));
```

Comment:
I am not sure what are you doing here by passing the alphabet. Looks weird

Reply:
In this case, I need to name an arbitrary city, so I use the alphabet to select a random letter and then select a random city from the list of cities for this letter